packages/components/src/autocomplete/src/components/osds-autocomplete/osds-autocomplete.scss

```diff
@@ -39,6 +39,14 @@
   animation: appear 0.2s ease-in-out;
 }
 
+:host(.ods-autocomplete) .anchor .overlay .loading {
```

Comment:
Can you use the nested implementation ?

Reply:
Yes it's a topic I wanted to cover once this one would be merged, I had issues with eslint using the nested, let's cover it in the other PR?